.github/workflows/erlang.yml

```diff
@@ -68,12 +47,15 @@ jobs:
       - uses: erlef/setup-beam@b9c58b0450cd832ccdb3c17cc156a47065d2114f # v1.18.1
         id: setup-beam
         with:
-          otp-version: ${{matrix.otp-version}}
-          rebar3-version: ${{matrix.rebar3-version}}
+          version-type: strict
+          version-file: .tool-versions
         if: ${{matrix.os-base != 'macos'}}
 
       - run: |
-          brew install erlang@${{matrix.otp-version}}
+          # there be dragons
+          brew update
+          erlang=$(< .tool-versions grep erlang | sed -E 's/erlang ([^\.]+).*/\1/g')
+          brew install "erlang@${erlang}"
           brew install rebar3
```

Comment:
Note that you should `brew link --overwrite --force "erlang@{erlang}"` to ensure that it's using the version of Erlang that you want.

Reply:
Hey, thanks for the heads up. 👍